dds/idl/langmap_generator.cpp

```diff
@@ -1474,7 +1474,9 @@ struct Cxx11Generator : GeneratorBase
       }
     } else {
       if (af.cls_ & CL_ARRAY) {
-        initializer = "{}";
+        // With array no initialize, gives a warning with gcc 4.8.5, for example
+        // warning: missing initializer for member ‘std::array<std::basic_string<char>, 5ul>::_M_elems’ [-Wmissing-field-initializers]
+        initializer = "";
```

Comment:
Does this change initialization for struct members that are arrays of fundamental types?

Reply:
Good catch, will check that and add a test for that explicitly